scripts/machine-launch-script.sh

```diff
@@ -1,83 +1,265 @@
 #!/bin/bash
 
+CONDA_INSTALL_PREFIX=/opt/conda
+CONDA_INSTALLER=https://github.com/conda-forge/miniforge/releases/download/4.11.0-4/Miniforge3-4.11.0-4-Linux-x86_64.sh
+CONDA_ENV_NAME="base"
+
+DRY_RUN_OPTION=""
+DRY_RUN_ECHO=()
+
+usage()
+{
+    echo "Usage: $0 [options]"
+    echo
+    echo "Options:"
+    echo "[--help]                  List this help"
+    echo "[--prefix <path>]         Install prefix for conda. Defaults to /opt/conda."
+    echo "                          If <path>/bin/conda already exists, it will be used."
```

Comment:
```suggestion
    echo "[--prefix <prefix>]         Install prefix for conda. Defaults to /opt/conda."
    echo "                          If <prefix>/bin/conda already exists, it will be used."
```